features/border-image.yml.dist

```diff
@@ -14,14 +14,87 @@ status:
     safari: "9.1"
     safari_ios: "9.3"
 compat_features:
-  - css.properties.border-image
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "15"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "13"
+  #   firefox_android: "14"
+  #   safari: "6"
+  #   safari_ios: "6"
+  - css.properties.border-image-width
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "15"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "15"
+  #   firefox_android: "15"
+  #   safari: "6"
+  #   safari_ios: "6"
```

Comment:
Ah this is cool! This is very likely a bug in the data, right? It seems deeply unlikely that Firefox shipped `border-image-width` before `border-image-source`.

Reply:
Yep. This grouping will throw up a lot of issues like this. If we try to fix them I'm sure we'll move slower instead of faster. Do we need some fast way to mark issues like this as future TODOs?